scipy/sparse/linalg/isolve/tests/test_lgmres.py

```diff
@@ -106,8 +106,8 @@ def test_arnoldi(self):
 
         assert_equal(flag0, 1)
         assert_equal(flag1, 1)
-        assert_(np.linalg.norm(A.dot(x0) - b) > 4e-4)
-
+        norm = np.linalg.norm(A.dot(x0) - b)
+        assert_(norm > 1e-4)
         assert_allclose(x0, x1)
```

Comment:
You can't see the norm currently. If this breaks again I'll want to know if norm is changing between runs.